### PR TITLE
PersonPicture update: Overwriteable properties for style

### LINKED ISSUE #10883: Make style properties of `PersonPicture` overwriteable
## What would you like to be added:
The background and other properties are not overwriteable, but this should normally be the case.
## Why is this needed:
Currently the default style have a fix background and other properties for the ellipse control.

## Anything else we need to know?
The issue is also active at WinUI2:
https://github.com/microsoft/microsoft-ui-xaml/issues/6654
https://github.com/microsoft/microsoft-ui-xaml/pull/6655

### PR DESCRIPTION
**Pull request by jp-weber**
GitHub Issue (If applicable): closes #10883

## PR Type

What kind of change does this PR introduce?

## What is the current behavior?

Currently the default style have a fix background and other properties for the ellipse control.

## What is the new behavior?

The background and other properties are now overwriteable.
grafik

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by Youssef1313 — APPROVED** [review]
Thanks @jp-weber 

I think we need to wait until https://github.com/microsoft/microsoft-ui-xaml/pull/6655 is merged first. Otherwise, LGTM

**Review by MartinZikmund — APPROVED** [review]

**Review by Youssef1313 — COMMENTED** [review]
Note: The corresponding WinUI PR currently has a test failure.
If changes are made in the other PR, they shouldn't be forgotten here.

**Review by ajpinedam — APPROVED** [review]

**Review by agneszitte-nventive — APPROVED** [review]

**Comment by Youssef1313** [comment]
PR https://github.com/microsoft/microsoft-ui-xaml/pull/6655 was merged

File: src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml [hunk 1]
```diff
@@ -74,9 +77,9 @@
                         </VisualStateManager.VisualStateGroups>
 
                         <Ellipse
-                            Fill="{ThemeResource PersonPictureEllipseFillThemeBrush}"
-                            Stroke="{ThemeResource PersonPictureEllipseFillStrokeBrush}"
-                            StrokeThickness="{ThemeResource PersonPictureEllipseStrokeThickness}"
+                            Fill="{TemplateBinding Background}"
+                            Stroke="{TemplateBinding BorderBrush}"
+                            StrokeThickness="{TemplateBinding BorderThickness}"
```
Comment (Youssef1313) [inline src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml:82]: Should we also update https://github.com/unoplatform/uno/blob/master/src/Uno.UI.FluentTheme.v1/Resources/Version1/PriorityDefault/PersonPicture.xaml ?

**Comment by MartinZikmund** [comment]
@jp-weber can you fix the conflict and check the comments above? After that this will be good to merge!

File: src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml (hunk 1 above)
Comment (jp-weber) [reply src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml:82]: 👍 I have updated version 1 accordingly.

**Comment by jp-weber** [comment]
@MartinZikmund 
I have now updated version 1 style according to the other changes.

**Comment by Youssef1313** [comment]
Thanks @jp-weber There is a merge conflict though, could you fix it please?

**Comment by jp-weber** [comment]
@Youssef1313 
I solved the conflict on my fork by opening the commit with GitHub desktop and pushing this without problems:
https://github.com/jp-weber/uno/commit/8338b10e5cd00994a5f14e205afac0d9353c54cd#diff-4d8d2e6a9dae022a62588061a98e168757fd64e051083a5779b0975d708b8edc
It seems the move of the folder 'Uno.UI/UI/Xaml' to 'Uno.UI/Microsoft/UI/Xaml' is causing problems which was auto fixed by GitHub Desktop. 

What is the best way to solve the problem here, an actual merge from master into my branch from commit?

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@jp-weber I've done a rebase on your branch, and it's been applied properly. Thanks for your contribution!

**Pull request merged**